### DISCUSSION TITLE
Identification in NomadNet and links

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#345] [General]
I have ticked the identify to server option and the environment does not contain "remote_identity" and the link number does not change. Does the link need to reset for it to identify itself? Using disconnect and reconnecting does not work. Restarting the server solved the issue, and the link ID changed. Is there a way to terminate the link to identify without cycling the server?

**Comment by markqvist** [comment]
You should be able to close the link by simply closing the connection in nomadnet, by pressing `Ctrl - W` in the browser window. Can you please check if that works?

**Reply by faragher** [reply]
Control - W works. I thought I tried that before, but it does work. Thanks.
